### DISCUSSION TITLE
Up one directory in context menu

### DISCUSSION THREAD

**Discussion by lyzanxia** [keirf/flashfloppy#661] [Ideas]
Sometimes I am in a directory with hundreds of files, when I am for example at file 360, I have a lot of scrolling to do to get back to ".." to go up a directory. Would be very handy if in the context menu there would be a "directory up" selection.

**Comment by lyzanxia** [comment]
As a sidenote: I forgot to tell that I don't have any buttons on my gotek (seller probably removed them - bought the Amiga as a package).

**Comment by keirf** [comment]
Shame about the lack of buttons as that's the obvious method. Perhaps I can hack it into the eject/context menu (actually perhaps context menu is a better name than eject menu).
